app/src/main/res/layout/walkthrough_activity.xml

```diff
@@ -42,6 +42,21 @@
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent" />
 
+    <TextView
+      android:id="@+id/walkthrough_activity_topic_header_text_view"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="@dimen/margin_16"
+      android:layout_marginTop="@dimen/margin_16"
+      android:fontFamily="sans-serif-medium"
+      android:text="@string/what_do_you_want_to_learn"
+      android:textColor="@color/oppiaPrimaryText"
+      android:textSize="24sp"
+      android:visibility="invisible"
+      app:layout_constraintBottom_toBottomOf="@+id/walkthrough_progress_bar"
```

Comment:
Does this look correct to you, bottom to the bottom or I am mistaking something here?

Reply:
Yes, this looks correct to me, alternatively, we can constraintBottom_toBottomOf = back_button (like ProgressBar does)

Reply:
I have resolved this